scanpy/external/exporting.py

```diff
@@ -211,11 +215,12 @@ def spring_project(
 # --------------------------------------------------------------------------------
 
 
-def _get_edges(adata):
+def _get_edges(adata, neighbors_key=None):
+    neighbors = NeighborsView(adata, neighbors_key)
     if 'distances' in adata.uns['neighbors']:  # these are sparse matrices
```

Comment:
Just ran into this. This shouldn't check for distances in `uns["neighbors"]`. Could this check if `"distances"` and `"connectivities"` are in `NeighborsView` instead?

Reply:
Yes, thank you for catching this.